releases/compiler/compiler.go

```diff
@@ -45,54 +45,56 @@ func (c *compiler) Compile(ctx context.Context, work Work) (string, error) {
 	goos := runtime.GOOS
```

Comment:
compiler.go had no outstanding problems other than some improvements to readability. If they are not pleasing or too specific to my own aesthetics i will revert this file no problemo.